library/Icinga/Crypt/RSA.php

```diff
@@ -0,0 +1,274 @@
+<?php
+/* Icinga Web 2 | (c) 2020 Icinga Development Team | GPLv2+ */
+
+namespace Icinga\Crypt;
+
+use InvalidArgumentException;
+use UnexpectedValueException;
+
+/**
+ * Data encryption and decryption using RSA keys also supporting key generation
+ *
+ * # Example Usage
```

Comment:
Example usage does not reflect our latest changes. The methods are no longer variadic.